types/webdriverio/index.d.ts

```diff
@@ -1,9 +1,10 @@
-// Type definitions for WebdriverIO 4.8
+// Type definitions for WebdriverIO 4.10
```

Comment:
just for reference: semver minor is the correct level. patch versions are omitted.